src/views/viewer/ViewerLayout.tsx

```diff
@@ -36,11 +39,22 @@ const viewerRoutes = [
   { path: relativeRoutes.viewer.channels(), element: <ChannelsView /> },
   { path: relativeRoutes.viewer.channel(), element: <ChannelView /> },
   { path: relativeRoutes.viewer.category(), element: <CategoryView /> },
+  {
+    path: relativeRoutes.viewer.editMembership(),
+    element: (
+      <PrivateRoute
+        isAuth={isActiveMember}
+        element={<EditMembershipView />}
+        redirectTo={absoluteRoutes.viewer.index()}
+      />
```

Comment:
in firefox I get this error if I go to the URL without login in `Unhandled exception was thrown: The operation is insecure. 
Object { cause: React ErrorBoundary Error }`

on Chrome it works fine.

Not sure if worth fixing or what is causing it

Reply:
hmm, I didn't do anything to fix this but I think it works now for some reason 🤔  Let me know if you manage to reproduce this!

Reply:
Is still the same for me but maybe is just my machine idk what it is

Reply:
Ok, I managed to reproduce this, but only once. Then I tried to do this with or without extension and with cleared local storage.  Everything seems to be fine. 
I assume that it's caused by changes inside ViewerLayout - I used PrivateRoute there and Navigate. 
Maybe I should revert that and we should all the private navigation in another PR?